### DISCUSSION TITLE
Git push does not work

### DISCUSSION THREAD

**Discussion by Imbrosci** [community/community#22049] [New to GitHub]
<p>I would like to upload to my remote repository some large file with git lfs. Opening git bash and performing all necessary steps is not a problem until the command git push. After enting my name and password I get an error. I read about the recent changes and the need to create a token. I did it and used it instead of my password but the problem is not solved. I still get an error. I have also updated git bash and performed some other updates I found to be reccomented in stack overflow:</p>
<p>git remote -v<br>
git remote remove origin<br>
git remote add origin <a href="mailto:git@github.com">git@github.com</a>:user/repo.git</p>
<p>but I still did not manage to get rid of errors. I think the translation of the last error from german should say something like:<br>
fatal: The request was interruped: no protected SLL/TLS-channel could be generated.</p>
<p>thanks for any suggestions</p>
<p>Barbara</p>

**Comment by byrneh** [comment, marked as answer]
<p>This command is telling git to use ssh protocol not https</p>
<blockquote>
<p>git remote add origin <a href="mailto:git@github.com">git@github.com</a>:user/repo.git</p>
</blockquote>
<p>If you want to try ssh follow the github instructions will alos explains how to create and add your needed ssh keys.</p>
<p>Using https make sure you have removed any cached credentials from local Credential manager . You should be prompted for credentials and enter your PAT not your password for successful authentication</p>

**Comment by Imbrosci** [comment]
<p>Thank you for the answer!</p>
